question09/main.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+// loading img 
+const wrapper = document.getElementById('wrapper');
+const loadImg = document.createElement('img');
+loadImg.src = "loading-circle.gif";
+wrapper.appendChild(loadImg);
```

Comment:
create function and insert `asyncFunc`.

Reply:
I made the function for showing Loading img and inserted to async function. Please confirm.

Latest commit : https://github.com/csakyo/homework/pull/19/commits/1d38e5e3ac222cc1073c6c89a38d3d015f77a2ef